src/connector.ts

```diff
@@ -17,6 +17,34 @@ import * as calvaDebug from './debugger/calva-debug';
 import { setStateValue, getStateValue } from '../out/cljs-lib/cljs-lib';
 import * as replSession from './nrepl/repl-session';
 import * as clojureDocs from './clojuredocs';
+import * as jszip from 'jszip';
+import { addEdnConfig } from './config';
+
+async function readJarContent(uri: string) {
+    console.log(uri);
```

Comment:
There's a `getJarContents` function in utilities.ts. I'm not sure if you knew that. Maybe it could be reused here?

Reply:
ah great, I didn't see that. I'll work it in later today, thanks for the feedback